openstates/tn/bills.py

```diff
@@ -229,13 +227,20 @@ def scrape_bill(self, term, bill_url, bill_type):
         # checking if there is a matching bill
         if secondary_bill_id:
             secondary_bill_id = secondary_bill_id[0].text
-
             # swap ids if * is in secondary_bill_id
             if '*' in secondary_bill_id:
                 bill_id, secondary_bill_id = secondary_bill_id, bill_id
                 secondary_bill_id = secondary_bill_id.strip()
-
-        bill_id = bill_id.replace('*', '').strip()
+            secondary_bill_id = secondary_bill_id.replace('  ',' ')
+            
+        bill_id = bill_id.replace('*', '').replace('  ',' ').strip()
+
+        if 'B' in bill_id:
+            bill_type = 'bill'
+        elif 'JR' in start_abbr:
+            bill_type = 'joint resolution'
+        else:
```

Comment:
May want to turn this `else` case into `elif 'R' in start_abbr`, and reserve the `else` case to emit an exception.  Although the assumption that anything else would be a resolution is largely good, it could potentially capture faulty data, and we would want to explicitly know if that's the case so that the issue can be investigated.